recipes-multimedia/libcamera/libcamera.bbappend

```diff
@@ -0,0 +1,5 @@
+SUMMARY = "Linux libcamera framework"
```

Comment:
Any reason to redefine this? (http://cgit.openembedded.org/meta-openembedded/tree/meta-multimedia/recipes-multimedia/libcamera/libcamera.bb?h=master#n1)